src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -559,10 +553,10 @@ private void searchForItems()
 
         //TODO check if sapling or apple (currently picks up all items, which may be okay)
         items = world.getEntitiesWithinAABB(EntityItem.class, worker.getEntityBoundingBox().expand(RANGE_HORIZONTAL_PICKUP, RANGE_VERTICAL_PICKUP, RANGE_HORIZONTAL_PICKUP))
-                .stream()
-                .filter(item -> item != null && !item.isDead)
-                .map(BlockPosUtil::fromEntity)
-                .collect(Collectors.toList());
+                     .stream()
+                     .filter(item -> item != null && !item.isDead)
+                     .map(BlockPosUtil::fromEntity)
+                     .collect(Collectors.toList());
     }
 
     /**
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)